releaseNotes/PythonFeatureManagement.md

```diff
@@ -2,6 +2,18 @@
 
 [Source code][source_code] | [Samples][samples]
 
+## 2.0.0 - January 6, 2025
+
+### Enhancements
+
+* Added support for variant feature flags. A variant feature flag is an enhanced feature flag that supports multiple states or variations. While it can still be toggled on or off, it also allows for different configurations, ranging from simple primitives to complex JSON objects. Variant feature flags are particularly useful for feature rollouts, configuration rollouts, and feature experimentation (also known as A/B testing).
+
+For more information, see the [feature reference document](https://learn.microsoft.com/azure/azure-app-configuration/feature-management-python-reference#variants).
+
+* Added support for telemetry in feature flags. Telemetry is a powerful feature that allows you to track how your feature flags are being used. It provides insights into the effectiveness of your feature flags and helps you make data-driven decisions. Telemetry is particularly useful for feature experimentation (also known as A/B testing) and feature rollouts.
```

Comment:
Telemetry ~is a powerful feature that~ allows you to track how your feature**s** ~flags~ are being used.

(I know we had this elsewhere but calling it powerful sounds too much :). Telemetry is for tracking features. The flags are just the means of doing it.)